it/controllers/OverviewPageControllerTest.scala

```diff
@@ -314,71 +489,72 @@ class OverviewPageControllerTest extends IntegrationTest with ViewHelpers {
             }
 
             "has an employment section" which {
-              linkCheck(employmentLinkText, employmentLinkSelector, employmentLink)
+              linkCheck(employmentLinkText, employmentLinkSelector, employmentLink(taxYear))
               textOnPageCheck(updatedText, employmentStatusSelector)
             }
 
             textOnPageCheck(viewTaxCalcText, viewTaxCalcSelector)
-            linkCheck(viewEstimateLinkText, viewEstimateSelector, viewEstimateLink)
+            linkCheck(viewEstimateLinkText, viewEstimateSelector, viewEstimateLink(taxYear))
             textOnPageCheck(submitReturnText, submitReturnSelector)
-            textOnPageCheck(youWillBeAbleIndividualText, youWillBeAbleSelector)
+            textOnPageCheck(specific.youWillBeAble(taxYearPlusOne), youWillBeAbleSelector)
           }
+        }
 
-          "interest income source is defined and the untaxed and taxed accounts do not have amounts" which {
-            val incomeSources = incomeSourcesModel.copy(interest = Some(Seq(InterestModel("TestName", "TestSource", None, None))))
-
-            val request = FakeRequest("GET", urlPath).withHeaders(headers: _*)
+        "interest income source is defined and the untaxed and taxed accounts do not have amounts" which {
+          val incomeSources = incomeSourcesModel.copy(interest = Some(Seq(InterestModel("TestName", "TestSource", None, None))))
 
-            lazy val result: Future[Result] = {
-              authoriseIndividual()
-              stubIncomeSources(incomeSources)
-              route(app, request).get
-            }
+          val request = FakeRequest("GET", urlPath).withHeaders(headers: _*)
 
-            implicit def document: () => Document = () => Jsoup.parse(Helpers.contentAsString(result))
+          lazy val result: Future[Result] = {
+            authoriseAgentOrIndividual(user.isAgent)
+            stubIncomeSources(incomeSources)
+            route(app, request).get
+          }
 
-            "returns status of OK(200)" in {
-              status(result) shouldBe OK
-            }
+          implicit def document: () => Document = () => Jsoup.parse(Helpers.contentAsString(result))
 
-            welshToggleCheck("English")
-            linkCheck(vcBreadcrumb, vcBreadcrumbSelector, vcBreadcrumbUrl)
-            linkCheck(startPageBreadcrumb, startPageBreadcrumbSelector, startPageBreadcrumbUrl)
-            textOnPageCheck(overviewBreadcrumb, overviewBreadcrumbSelector)
+          "returns status of OK(200)" in {
+            status(result) shouldBe OK
+          }
 
-            titleCheck(individualHeading)
-            h1Check(individualHeading, "xl")
-            textOnPageCheck(caption, captionSelector)
-            textOnPageCheck(updateIncomeTaxReturnTextIndividual, dividendsProvideUpdatesSelector)
-            textOnPageCheck(completeSectionsText, completeSectionsSelector)
+          welshToggleCheck("English")
```

Comment:
same as above, since you've updated to the new pattern here should update to use user.isWelsh for all test cases